packages/quantic/force-app/main/default/lwc/quanticNumericFacet/quanticNumericFacet.js

```diff
@@ -102,17 +122,47 @@ export default class QuanticNumericFacet extends LightningElement {
 
   /** @type {NumericFacetState} */
   @track state;
+  /** @type {NumericFilterState} */
+  @track stateFilter = {
+    isLoading: false,
+    facetId: undefined
+  }
 
   /** @type {NumericFacet} */
   facet;
-  /** @type {Function} */
+  /**  @type {import("coveo").NumericFilter} */
+  numericFilter;
+  /**  @type {import("coveo").SearchStatus} */
+  searchStatus;
+  /** @type {import("coveo").Unsubscribe} */
```

Comment:
```suggestion
  /** @type {Function} */
```
This type import resolves to `any` so Function is better and works better with doc gen